Fix some typos (#379)
atomisthq/docs: docs/integration/deploys.md

## Tracking Image Deployments

Atomist compares recent image vulnerabilty scans against the scans of images that are currently deployed.

This level of tracking gives developers contexts about when security debt is both increasing and decreasing.  This can be integrated in several ways:

* steps in continuous deployment pipelines
* Kubernetes Admission Controllers (for kube-based deployments)

> * calling API directly

Reviewer: ```suggestion
* Calling the API directly
```

## Api

> Each Atomist workspace exposes an endpoint that can be called whenever an image is deployed.  All subsequent scans will now be compared against each recorded environment.  Lookup the endpoint URL in the [Atomist app](https://dso.atomist.com/r/auth/integrations).

Reviewer: ```suggestion
Each Atomist workspace exposes an endpoint to call whenever an image is deployed.  All subsequent scans will now be compared against each recorded environment.  Look up the endpoint URL in the [Atomist app](https://dso.atomist.com/r/auth/integrations).
```

![integration](img/deploy/integration.png)

!!! Note
    you must create an api-key to use this endpoint.

> The simplest form of interation is to call this API whenever a new image is deployed.  The default environment name is `deployed` - if you want to image scans, and GitHub Checks to start comparing against a "deployed" image, then call:

Reviewer: ```suggestion
The simplest form of integration is to call this API whenever a new image is deployed.  The default environment name is `deployed` - if you want to scan images, and activate GitHub Checks to start comparing against a "deployed" image, then call:
```
